examples/tutorials/mvdr_tutorial.py

```diff
@@ -38,258 +51,304 @@
 #
 #    ``SSB07200001\#noise-sound-bible-0038\#7.86_6.16_3.00_3.14_4.84_134.5285_191.7899_0.4735\#15217\#25.16333303751458\#0.2101221178590021.wav``
 #
-# which was generated with;
+# , which was generated with:
 #
-# - ``SSB07200001.wav`` from `AISHELL-3 <https://www.openslr.org/93/>`__ (Apache License v.2.0)
-# - ``noise-sound-bible-0038.wav`` from `MUSAN <http://www.openslr.org/17/>`__ (Attribution 4.0 International — CC BY 4.0)  # noqa: E501
+# -  ``SSB07200001.wav`` from
+#    `AISHELL-3 <https://www.openslr.org/93/>`__ (Apache License
+#    v.2.0)
+# -  ``noise-sound-bible-0038.wav`` from
+#    `MUSAN <http://www.openslr.org/17/>`__ (Attribution 4.0
+#    International — CC BY 4.0)
 #
 
-import os
+import matplotlib.pyplot as plt
+from IPython.display import Audio
 
-import IPython.display as ipd
-import requests
-import torch
-import torchaudio
+from torchaudio.utils import download_asset
 
-torch.random.manual_seed(0)
-device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+SAMPLE_RATE = 16000
+SAMPLE_MIX = download_asset("tutorial-assets/mvdr/mix.wav")
+SAMPLE_CLEAN = download_asset("tutorial-assets/mvdr/reverb_clean.wav")
 
-print(torch.__version__)
-print(torchaudio.__version__)
-print(device)
 
-filenames = [
-    "mix.wav",
-    "reverb_clean.wav",
-    "clean.wav",
-]
-base_url = "https://download.pytorch.org/torchaudio/tutorial-assets/mvdr"
+######################################################################
+# 2.1. Helper functions
+# ~~~~~~~~~~~~~~~~~~~~~
+#
+
+
+def plot_spectrogram(stft, title="Spectrogram", xlim=None):
+    magnitude = stft.abs()
+    spectrogram = 20 * torch.log10(magnitude + 1e-8).numpy()
+    figure, axis = plt.subplots(1, 1)
+    img = axis.imshow(spectrogram, cmap="nipy_spectral", vmin=-100, vmax=0, origin="lower", aspect="auto")
```

Comment:
The custom colormap is cool, but can we use the default `viridis`?
It is designed so that it is easy for people with color blindness to understand the plot.

See the story behind of the development https://bids.github.io/colormap/